web-animations/interfaces/Animation/play.html

```diff
@@ -31,5 +31,33 @@
 }, 'play() throws when seeking an infinite-duration animation played in ' +
    'reverse');
 
+test(function(t) {
+  var div = createDiv(t);
+  var animation = div.animate(null, 100 * MS_PER_SEC);
+  animation.currentTime = 10 * MS_PER_SEC;
+  assert_times_equal(animation.currentTime, 1 * MS_PER_SEC);
```

Comment:
I meant to change it to 1 * MS_PER_SEC to make it harder to confuse with 100 * MS_PER_SEC.